modules/stanford_profile_styles/lib/scss/paragraph-types/react_paragraphs/_multirow.scss

```diff
@@ -0,0 +1,144 @@
+@charset 'UTF-8';
+
+// For full width pages
+.jumpstart-ui--one-column {
+  .react-paragraphs-row {
+
+    // Setting the max width to help with display and
+    // to keep the margins on full-width pages.
+    .paragraph-item {
+
+      // 4 items across a page.
+      &.flex-3-of-12 {
+        &.ptype-stanford-banner {
+          @include grid-media-between('xs', 'md') {
+            min-width: 100%
+          }
+        }
+
+        &.ptype-stanford-card,
+        &.ptype-stanford-wysiwyg,
+        &.ptype-stanford-media-caption {
+          @include grid-media('xl') {
+            @include margin(null $su-gutter-xl);
+            max-width: 20%;
+          }
+
+          @include grid-media-between('lg', 'xl') {
+            @include margin(null $su-gutter-lg);
+            max-width: 18%;
+          }
+        }
+      }
+
+      // Three items across a page.
+      &.flex-4-of-12 {
+        @include grid-media-between('lg','xl') {
+          @include margin(null 0);
+          max-width: 33%;
+        }
+
+        @include grid-media-between('xs', 'md') {
+          max-width: 100%;
+        }
+
+        &.ptype-stanford-banner {
+          @include grid-media-between('xs', 'md') {
+            min-width: 100%
+          }
+        }
+
+        &.ptype-stanford-card,
+        &.ptype-stanford-wysiwyg,
+        &.ptype-stanford-media-caption {
+          @include grid-media('xl') {
+            @include margin(null $su-gutter-xl);
+            max-width: 28%;
+          }
+
+          @include grid-media-between('lg', 'xl') {
+            @include margin(null $su-gutter-lg);
+            max-width: 26%;
+          }
+        }
+      }
+
+      // 2 items across a page.
+      &.flex-6-of-12 {
+        &.ptype-stanford-banner {
+          @include grid-media-between('xs', 'md') {
+            min-width: 100%
+          }
+        }
+        &.ptype-stanford-card,
+        &.ptype-stanford-wysiwyg,
+        &.ptype-stanford-media-caption {
+          @include grid-media('xl') {
+            @include margin(null $su-gutter-xl);
+            max-width: 45%;
+          }
+          @include grid-media-between('lg','xl') {
+            @include margin(null $su-gutter-lg);
+            max-width: 43%;
+          }
+        }
+
+        @include grid-media('lg') {
+          @include margin(null 0);
+          max-width: 50%;
+        }
+
+        @include grid-media-between('xs', 'md') {
+          max-width: 100%;
+        }
+      }
+    }
+  }
+}
+
+// For all templates.
+.react-paragraphs-row {
+
+  // Single item per row on screens less than and equal to md.
+  &.flex-container {
+    @include grid-media-max('md') {
+      display: block;
+      .paragraph-item {
+        max-width: 100%;
+      }
+    }
+  }
```

Comment:
This is a little more generic and flatter than the previous jumpstart-ui-two-column class selector but accomplished the same thing of setting the display to block for screens smaller than (md)